src/components/SelectableCollection/SelectableCollection.module.scss

```diff
@@ -1,6 +1,5 @@
-@import 'styles/uswdsDependencies';
-@import 'styles/sfds/sfdsDependencies';
-@import 'styles/deps';
```

Comment:
Do we need to add `@use 'styles/deps'` here?

Reply:
`styles/deps` contains button reset, which isn't used in this file, so we don't need it 👍